src/Hooks.tsx

```diff
@@ -1,8 +1,10 @@
 import { useEffect, useState } from 'react'
 
-function useLocalStorage<T>(key: string, initialValue: T): [T, (v: T) => void] {
+function useStorage<T>(key: string, initialValue: T, type: 'session' | 'local'): [T, (v: T) => void] {
```

Comment:
I think `useBrowserStorage` might be clearer?

Reply:
agreed. Fixed